lightningd/closing_control.c

```diff
@@ -52,16 +52,36 @@ struct close_command {
 /* Resolve a single close command. */
 static void
 resolve_one_close_command(struct close_command *cc, bool cooperative,
-			  const struct bitcoin_tx *close_tx)
+			  struct bitcoin_tx **close_txs)
 {
+	assert(tal_count(close_txs));
 	struct json_stream *result = json_stream_success(cc->cmd);
+	const struct bitcoin_tx *close_tx = close_txs[tal_count(close_txs) - 1];
 
 	json_add_tx(result, "tx", close_tx);
 	if (!invalid_last_tx(close_tx)) {
 		struct bitcoin_txid txid;
 		bitcoin_txid(close_tx, &txid);
 		json_add_txid(result, "txid", &txid);
```

Comment:
nit: deprecate the `txid` field, in favor of just using the `txids` field in the future.

Reply:
I deprecated `txid` and `tx`

Reply:
ah ok, missed it the first time! seems like the deprecation feature stuff got updated, very cool.